docs/conf.py

```diff
@@ -32,10 +32,13 @@
     "sphinx.ext.napoleon",
     "IPython.sphinxext.ipython_console_highlighting",
     #"myst_nb",
+    "myst_parser",
```

Comment:
Do not add `myst_parser` to conf.py because it causes error in Sphinx build, see commits under this comment:
https://github.com/r5py/r5py/pull/38#pullrequestreview-969167018

Reply:
It's weird because Sphinx documentation tells to do this, but apparently these are taken care of by `sphinx_book_theme`.

Reply:
Hmm, I take these comments back. It seems that `myst_nb` was taking care of these, but now as we are not using it, the `myst_parser` and `source_suffix` seem to be required.